### DISCUSSION TITLE
老版本 v4 `forward` 模块支持 `enable_pipeline` 的吗？

### DISCUSSION THREAD

**Discussion by sbwml** [IrineSistiana/mosdns#647] [Q&A] [answered]
RT，@urlesistiana 大佬

https://irine-sistiana.gitbook.io/mosdns-wiki/mosdns-v4/cha-jian-ji-qi-can-shu#forward-zhuan-fa-qing-qiu-zhi-shang-you-dnsproxy

在docs 中未见 `enable_pipeline` 参数示例，这个是仅存在于 `fast_forward` 模块中的吗？

**Comment by urlesistiana** [comment, marked as answer]
不支持。dnsproxy 没有实现 pipeline。

**Comment by sbwml** [comment]
谢谢大佬回复，另外 `fast_forward` 模块的 bootstrap 说明有提到：bootstrap: (v4.2+)(实验性) 手动指定用于解析 addr 中域名的 bootstrap 服务器。只能是 UDP 服务器。指定后插件**每次连接服务器时**会先通过该 bootstrap 服务器解析域名。

这个**每次连接服务器时** 是无法打破的吗？它能否实现和 `forward` 模块一样只在启动时**解析一次**的？

**Reply by urlesistiana** [reply]
> 这个每次连接服务器时 是无法打破的吗？它能否实现和 forward 模块一样只在启动时解析一次的？

不能。

用 v5。v5 这些都优化了。

**Reply by sbwml** [reply]
了解
